test/pubSubStats.js

```diff
@@ -273,9 +273,9 @@ describe('test/pubSubStats.js >', () => {
       const re1 = /info: activity=pubsub key=bye.world process=MyProcessName subCount=3 subTime=\d+ emitCount=3 emitTime=\d+ clientCount=3 clientTime=\d+ \n/; // jscs:ignore maximumLineLength
       const re2 = /info: activity=pubsub key=hello.world process=MyProcessName subCount=1 subTime=\d+ emitCount=1 emitTime=\d+ clientCount=1 clientTime=\d+ \n/; // jscs:ignore maximumLineLength
       expect(inspect.output).to.be.an('Array');
-      expect(inspect.output).to.have.lengthOf(2);
-      expect(inspect.output[0]).to.match(re1);
-      expect(inspect.output[1]).to.match(re2);
```

Comment:
Can you explain why the logging is doubled? Is there a way to prevent that so we don't have to change these tests?

Reply:
Yea sure, there is just a extra log line that is printed when local logging is on. Will change it.